src/jit/lir.cpp

```diff
@@ -916,6 +1038,12 @@ bool LIR::Range::CheckLIR(Compiler* compiler) const
         {
             if (unusedDefs.Index(i) == def)
             {
+                // REVIEW: isn't this the reverse? Shouldn't it be:
+                // for (; i < height - 1; i++) {
+                //    unusedDefs.IndexRef(i) = unusedDefs.Index(i + 1);
+                // }
+                // unusedDefs.Pop();
+                // --- better, just add an unusedDefs.Remove(i) member function.
```

Comment:
I plan on changing this code to just use `SmallHashTable` now that the LSRA changes have landed :)